feature-group-definitions/http11.yml

```diff
@@ -0,0 +1,77 @@
+name: HTTP/1.1
+spec: https://datatracker.ietf.org/doc/html/rfc2616
+legacy: true
```

Comment:
As discussed in #623 how about `baseline: high` and just no `baseline_low_date`?
Or using a keyword like `baseline_low_date: browser_set_start`?

The word `legacy` makes it feel, to me, like this is a feature people shouldn't use anymore.

Reply:
I've replied on that issue.

How about calling the features archived instead of legacy? It's something we've done to them rather than a property of the feature itself.

Reply:
"Archived" is better but I guess we still would need to make clear that it is not a property of the feature itself as HTTP1.1 is not archived. For example, AppCache was deprecated, then removed and I guess could now be "archived".

Maybe "legacy" or "archived" are terms that consumers might decide to apply themselves rather than web-features coming up with a name and rules for it here? Different consumers might have different rules when they decide to expose feature groups and when not.
 
MDN probably wants to display "baseline: high" for HTTP/1.1 features, so I'm assuming it shouldn't ignore or do anything special with this "archived" group, right?

Reply:
Features like AppCache should be removed from BCD and web-features when they're no longer part of the platform. While features which are "deprecated" only in the sense that their use is discouraged would have a `deprecated` status, but Baseline and web-features don't handle this case yet.

Both situations are orthogonal to a feature being archived.

I would expect MDN to do nothing with the archived flag since they do not expose the features directly.

Additional archived features that could make sense are CSS 2, HTML 4 and maybe the oldest parts of SVG.

The most important consumer for this is caniuse, and I'd recommend @Fyrd to not import archived features.

Reply:
I've made the change to `archived`. I won't be around much in the coming two weeks, and I guess this has a better chance of being merged with that change.